backend_py/libs/services/src/webviz_services/sumo_access/parameter_access.py

```diff
@@ -1,5 +1,5 @@
 import logging
-from typing import List, Optional
+from typing import List, Optional, Tuple
```

Comment:
We can just use the built in `list` and `tuple`, which is best practice for the Python versions we use. 

We can also replace `Optional` with ` | None` notation, i.e. `str | None` for strings.